site/server/svgPngExport.tsx

```diff
@@ -1,13 +1,26 @@
 import * as fs from "fs-extra"
 import * as sharp from "sharp"
 import * as path from "path"
+import * as svgo from "svgo"
 
 declare var global: any
 global.window = { location: { search: "" } }
 global.App = { isEditor: false }
 
 import { ChartConfig, ChartConfigProps } from "charts/ChartConfig"
 
+const svgoConfig: svgo.Options = {
+    floatPrecision: 1,
+    plugins: [{ collapseGroups: false }, { removeViewBox: false }]
+}
```

Comment:
This config is maybe up to some debate, and its worth noting that it extends the [default config](https://github.com/svg/svgo/blob/master/.svgo.yml).